src/Phinx/Migration/Manager.php

```diff
@@ -555,7 +609,7 @@ public function setMigrations(array $migrations)
      * Gets an array of the database migrations.
      *
      * @throws \InvalidArgumentException
-     * @return AbstractMigration[]
+     * @return AbstractMigration[], indexed by migration name (aka creation time) and sorted in ascending order
```

Comment:
so where should the `Indexed by migration name (aka creation time) and sorted in ascending order` part go to? I thought that for clarity it's important that the structure of that's method's returned array is defined somewhere